test/e2e/specs/editor/various/list-view.spec.js

```diff
@@ -0,0 +1,345 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+const openListView = async ( { page, pressKeyWithModifier } ) => {
+	// Open List View with keyboard.
+	await pressKeyWithModifier( 'access', 'o' );
+	// Wait until the current item is selected to ensure UI is ready.
+	await page
+		.getByRole( 'gridcell', {
+			name: /link/i,
+			selected: true,
+		} )
+		.waitFor();
+};
```

Comment:
> The second waitFor call is usually not needed in Playwright. Actions will auto-wait for the UI to be ready.

I agree it's usually the case, although this `waitFor` is needed in case the `openListView` is immediately followed by another `keypress` (like [here](https://github.com/WordPress/gutenberg/blob/e91096e4c177dae74bf55eb8f699bc0b819ca508/test/e2e/specs/editor/various/list-view.spec.js#L79)), which can kick in before the List View is opened. Does that make sense?

Reply:
Oh yeah, that makes sense! I'd still inline it wherever it's needed though, but that's just my opinion and yours works too!

Reply:
Ah, I actually misunderstood what you meant by inlining - I thought you meant to drop the `waitFor` entirely. Yeah, I was actually on the fence about creating that helper. I think I like inlined version better too, so I'll just revert that commit where I created it! 😄

Reply:
Updated in https://github.com/WordPress/gutenberg/pull/47919/commits/80a52315cee116a768951862197552aa3809f1c5.